### ISSUE TITLE
Total Duration time is incorrect

### ISSUE THREAD

**Issue by xxhhmxx** [jenkinsci/cucumber-reports-plugin#451]
### Jenkins and plugins versions report

<details>
  <summary>Environment</summary>

  
  ```text
  Paste the output here
  ```

</details>

### What Operating System are you using (both controller, and any agents involved in the problem)?

Linux

### Reproduction steps

![image](https://github.com/jenkinsci/cucumber-reports-plugin/assets/15605223/e1bf6a4f-be46-4035-8c20-9665c147ca00)

### Expected Results

Should sum all time above

### Actual Results

Missing hour number in total time

**Comment by damianszczepanik** [comment]
Attach JSON file

**Comment by damianszczepanik** [comment]
Is it because JSON report contains **background** elements which are not counted?

**Comment by jgrammen-agilitypr** [comment]
30.507 + 37.123 + 1:57.9 + 1:9.265 = 4:14.795, so the addition is correct but there is a discrepancy between the time reported by jenkins and the time reported by cucumber? maybe because there is (in my case 3) test cases running in parallel?

**Comment by damianszczepanik** [comment]
Yes. It just sums what is in JSON file, nothing else. If you have background elements or parallel then the sum might not be so clear
